src/main/java/io/jenkins/plugins/orka/OrkaCloud.java

```diff
@@ -57,6 +59,7 @@ public OrkaCloud(String name, String credentialsId, String endpoint, List<? exte
 
         this.mappings = mappings;
         this.templates = templates == null ? Collections.emptyList() : templates;
+        this.sshUtil = new SSHUtil();
```

Comment:
Ctors are not executed during deserialization, which happens on restart of the Jenkins server.
This means that when someone updates the plugin with this version, the ctor will not be called, `sshUtil` will be null an a null ref will be thrown.
A way to fix that is to use the `readResolve` method. It will be called during serialization.

Having said that, I do not see a reason for using an instance method. `waitForSSH` can be static. It will also eliminate the need to initialize the util at the correct place.